src/app/components/core/configuration-manager/kv-list-type-configuration/kv-list-type-configuration.component.html

```diff
@@ -0,0 +1,48 @@
+<div [formGroup]="kvListItemsForm">
+  <div formArrayName="kvListItems">
+    <ng-container *ngFor="let item of kvListItems.controls; let i=index">
+      <div class="columns mb-0 is-variable is-1" [formGroupName]="i">
+        <div class="column is-5">
+          <input formControlName="key" placeholder="key" class="input is-small" name="type" type="text">
+        </div>
+        <div class="column is-6">
+          <input *ngIf="configuration.items == 'integer'" class="input is-small" type="text" formControlName="value"
+            [minValue]="configuration?.minimum" [maxValue]="configuration?.maximum" appIntegerOnly>
+          <input *ngIf="configuration.items == 'float'" class="input is-small" type="text" formControlName="value"
+            [minValue]="configuration?.minimum" [maxValue]="configuration?.maximum" appFloatOnly>
+          <input *ngIf="configuration.items == 'string'" class="input is-small" type="text" formControlName="value"
+            [attr.maxLength]="configuration?.length">
+          <input *ngIf="configuration.items == 'enumeration'" class="input is-small" type="text" formControlName="value"
```

Comment:
enum is handled? We should add a graceful handling with TODO: FOG-x for Object type items